data-loader/cli/src/main/java/com/scalar/db/dataloader/cli/util/FileUtils.java

```diff
@@ -0,0 +1,37 @@
+package com.scalar.db.dataloader.cli.util;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.apache.commons.lang3.StringUtils;
+
+public class FileUtils {
+
+  /**
+   * Validates the provided file path.
+   *
+   * @param filePath the file path to validate
+   * @throws InvalidFilePathException if the file path is invalid
+   */
+  public static void validateFilePath(String filePath) throws InvalidFilePathException {
+    if (StringUtils.isBlank(filePath)) {
+      throw new IllegalArgumentException("File path must not be blank.");
+    }
+    Path pathToCheck = Paths.get(filePath);
+
+    if (!pathToCheck.isAbsolute()) {
+      // If the path is not absolute, it's either a file name or a relative path
+      Path currentDirectory = Paths.get("").toAbsolutePath();
+      Path fileInCurrentDirectory = currentDirectory.resolve(pathToCheck);
```

Comment:
It's not sufficient to use only `pathToCheck` without `currentDirectory`?

Reply:
Yes, only need to check `pathToCheck` exists. The specific condition is not required. I had added this in an early phase where I tried testing with a file name passed as an argument and that scenario is no longer required.
I have removed this part.
Thank you.